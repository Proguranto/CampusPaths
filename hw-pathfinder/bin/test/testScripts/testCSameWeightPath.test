# Find a path with Dijkstra's algorithm when
# the graph is cyclic and there are multiple paths
# from start to dest node with same edge weights.

CreateGraph graph7

AddNode graph7 S
AddNode graph7 1
AddNode graph7 2
AddNode graph7 D

AddEdge graph7 S 1 0.0
AddEdge graph7 1 2 0.0
AddEdge graph7 2 S 0.0
AddEdge graph7 S D 1.0

FindPath graph7 S D
