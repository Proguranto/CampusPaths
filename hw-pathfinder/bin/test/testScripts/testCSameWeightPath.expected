# Find a path with Dijkstra's algorithm when
# the graph is cyclic and there are multiple paths
# from start to dest node with same edge weights.

created graph graph7

added node S to graph7
added node 1 to graph7
added node 2 to graph7
added node D to graph7

added edge 0.000 from S to 1 in graph7
added edge 0.000 from 1 to 2 in graph7
added edge 0.000 from 2 to S in graph7
added edge 1.000 from S to D in graph7

path from S to D:
S to D with weight 1.000
total cost: 1.000
